tests/python/unittest/test_tvmscript_type.py

```diff
@@ -173,7 +173,7 @@ def different_access_indices(a: T.handle, b: T.handle) -> None:
                     ]
                 )
                 with T.init():
-                    B[vj, vi] = T.exp(B[vi, vj])
+                    B[vj, vi] = T.exp(B[vj, vi], dtype="float32")
```

Comment:
Does it mean that we no longer allow T.exp without dtype explicitly set? If not, what's the default dtype for this?